scripts/fast-reboot

```diff
@@ -410,6 +410,12 @@ docker exec -i bgp pkill -9 zebra
 docker exec -i bgp pkill -9 bgpd || [ $? == 1 ]
 debug "Stopped  bgp ..."
 
+# Kill nat docker after saving the conntrack table
+debug "Stopping nat ..."
+/usr/bin/dump_nat_entries.py
```

Comment:
Any reason to put this after stopping radv, bgp ? If we stop before it will have no impact on bgp restart timeout

Reply:
No specific reason. moved it to before radv.